docs/api/ref/PrivateChannel.md

```diff
@@ -0,0 +1,144 @@
+---
+id: PrivateChannel
+sidebar_label: PrivateChannel
+title: PrivateChannel
+hide_title: true
+---
+# `PrivateChannel`
+
+Object representing a private context channel, which is intended to support secure communication between applications, and extends the Channel interface with event handlers which provide information on the connection state of both parties, ensuring that desktop agents do not need to queue or retain messages that are broadcast before a context listener is added and that applications are able to stop broadcasting messages when the other party has disconnected.
+
+It is intended that Desktop Agent implementations:
+- SHOULD restrict external apps from listening or publishing on this channel.
+- MUST prevent private channels from being retrieved via fdc3.getOrCreateChannel.
+- MUST provide the `id` value for the channel as required by the Channel interface.
+
+```ts
+interface  PrivateChannel extends Channel {
+  // methods
+  onAddContextListener(handler: (contextType?: string) => void): Listener;
+  onUnsubscribe(handler: (contextType?: string) => void): Listener;
+  onDisconnect(handler: () => void): Listener;
+  disconnect(): void;
+}
+```
+
+#### See also
+
+* [`Channel`](Channel)
+* [`Listener`](Types#listener)
+* [`DesktopAgent.addIntentListener`](DesktopAgent#addintentlistener)
+* [`DesktopAgent.createPrivateChannel`](DesktopAgent#createPrivateChannel)
+* [`DesktopAgent.raiseIntent`](DesktopAgent#raiseintent)
+
+## Examples
+### 'Server-side' example:
+The intent app establishes and returns a PrivateChannel to the client (who is awaiting `getResult()`). When the client calls `addContextlistener()` on that channel, the intent app receives notice via the handler added with `onAddContextListener()` and knows that the client is ready to start receiving quotes.
+
+The Desktop Agent knows that a channel is being returned by inspecting the object returned from the handler (e.g. check constructor or look for private member).
+
+```javascript
+fdc3.addIntentListener("QuoteStream", async (context) => {
+	const channel: PrivateChannel = await fdc3.createPrivateChannel();
+	const symbol = context.id.ticker;
+
+	// This gets called when the remote side adds a context listener
+	const addContextListener = channel.onAddContextListener((contextType) => {
+		// broadcast price quotes as they come in from our quote feed
+		feed.onQuote(symbol, (price) => {
+			channel.broadcast({ type: "price", price});
+		});
+	});
+
+	// This gets called when the remote side calls Listener.unsubscribe()
+	const unsubscriberListener = channel.onUnsubscribe((contextType) => {
+		feed.stop(symbol);
+	});
+
+	// This gets called if the remote side closes
+	const disconnectListener = channel.onDisconnect(() => {
+		feed.stop(symbol);
+	});
+
+	return channel;
+});
+```
+
+### 'Client-side' example:
+
+```javascript
```

Comment:
```suggestion
### 'Client-side' example:
The 'client' application retrieves a `Channel` by raising an intent with context and awaiting the result. It adds a `ContextListener` so that it can receive messages from it. If a `PrivateChannel` was returned this may in turn trigger a handler added on the 'server-side' with `onAddContextListener()` and start the stream. A listener may also be to clear up if the 'server-side' disconnects from the stream. 

Although this interaction occurs entirely in frontend code, we refer to it as the 'client-side' interaction as it requests and receives a stream of responses.
```javascript
```